netlogo-gui/src/test/app/infotab/InfoFormatterTests.scala

```diff
@@ -104,13 +105,13 @@ class InfoFormatterTests extends FunSuite {
 
   // in contrast, here no <pre> because no blank line
   test("fenced code blocks 2") {
-    assertResult("""|<p>foo<br/><code>
-              |   bar
-              |  baz
-              |qux
-              |</code><br/>oof</p>""".stripMargin) {
+    assertResult("""|<p>foo<br/><code class="text">
```

Comment:
I think the purpose of this test is to clarify that code blocks appearing within a markdown paragraph get rendered with the same `<p>` in HTML as the surrounding text. I don't believe this is important because `<p>` isn't really semantic. We don't name any of the `<p>` so it isn't as though users can link to them. So the question remaining is whether this is a style concern. I've gone through and checked the models library and there are no models (that I could find) which relied on this old behavior (only a few models used backtick-delimited code sections) so that isn't a concern either. If there *are* display problems, we should be fixing them by altering CSS, not by using the `<pre>` tag.

So in summary, yes, let's delete the test!